cellprofiler/measurements.py

```diff
@@ -1859,8 +1859,6 @@ def load_measurements(filename, dest_file = None, can_overwrite = False,
             m = Measurements(filename=dest_file, copy = top_level,
                              image_numbers = image_numbers)
             return m
-        except:
```

Comment:
If you look at this closely, you'll see it swallows the exception and returns None... and then later, None.get_relationship_whatever() fails.